Error compiling with platformio for xiao c6

I have followed a previous guide on how to compile nimble  fir xiao c6 with platformio and I using the following env configuration:
```
 [env:t_bridge_xiao_esp32c6]
  platform = https://github.com/Jason2866/platform-espressif32.git#133f1115feb32983a1a11ba716f72bce982a040d
  board = xiao_esp32c6
  framework = arduino
  lib_deps =
	ricaun/ArduinoUniqueID@^1.3.0
	sstaub/TickTwo@^4.4.0
        h2zero/esp-nimble-cpp@^2.0.2
```
With the above I get this error:
```
.pio/libdeps/t_bridge_xiao_esp32c6/esp-nimble-cpp/src/NimBLEUtils.cpp:40:43: error: 'CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT' was not declared in this scope
```
I don;t know what what is expected for that define but if I define any value for CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT the compile/link works. I added this line to the platform.ini configuration:
```
build_flags = -D CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT=1
```
Are we suppose to give a value for CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT? If so, what should it be?

Thanks,
Humberto

--- a/library.json
+++ b/library.json
@@ -1,6 +1,6 @@
 {
   "name": "esp-nimble-cpp",
-  "version": "2.0.1",
+  "version": "2.0.2",
   "description": "C++ wrapper for the NimBLE BLE stack",
   "keywords": [
     "BLE",

--- a/src/NimBLEUtils.cpp
+++ b/src/NimBLEUtils.cpp
@@ -37,6 +37,9 @@
 # include <climits>
 
 # if defined INC_FREERTOS_H
+#  ifndef CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT
+#   define CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT 31
+#  endif
 constexpr uint32_t TASK_BLOCK_BIT = (1 << CONFIG_NIMBLE_CPP_FREERTOS_TASK_BLOCK_BIT);
 # endif
 

